.github/workflows/pull-request.yml

```diff
@@ -44,10 +44,10 @@ jobs:
           
           brew install sonar-scanner
           
-          bundle exec fastlane test scheme:"OneLogin" \
+          bundle exec fastlane test scheme:"OneLoginStaging" \
```

Comment:
Do we need to change the name of this job? It's currently "Build and Test **default** scheme"

Reply:
Good shout, will make changes to the names of some of these actions, they could be a lot clearer